src/schema/resolvers.js

```diff
@@ -112,63 +118,71 @@ function buildVoteFilters({ OR = [], oracleAddress, voterAddress, voterQAddress,
     },
 
-    allOracles: async (root, { filter, orderBy, limit, skip }, {db: {Oracles}}) => {
-      let query = filter ? {$or: buildOracleFilters(filter)}: {};
-      var cursor = Oracles.cfind(query);
+    allOracles: async (root, {
+      filter, orderBy, limit, skip,
+    }, { db: { Oracles } }) => {
+      const query = filter ? { $or: buildOracleFilters(filter) } : {};
+      let cursor = Oracles.cfind(query);
       cursor = buildCursorOptions(cursor, orderBy, limit, skip);
-      return await cursor.exec();
+      return cursor.exec();
     },
 
-    searchOracles: async (root, { searchPhrase, orderBy, limit, skip }, {db: {Oracles}}) => {
-      let query = searchPhrase ? {$or: buildSearchOracleFilter(searchPhrase)}: {};
-      var cursor = Oracles.cfind(query);
+    searchOracles: async (root, {
+      searchPhrase, orderBy, limit, skip,
+    }, { db: { Oracles } }) => {
+      const query = searchPhrase ? { $or: buildSearchOracleFilter(searchPhrase) } : {};
+      let cursor = Oracles.cfind(query);
       cursor = buildCursorOptions(cursor, orderBy, limit, skip);
-      return await cursor.exec();
+      return cursor.exec();
     },
 
-    allVotes: async (root, { filter, orderBy, limit, skip }, {db: {Votes}}) => {
-      let query = filter ? {$or: buildVoteFilters(filter)}: {};
-      var cursor = Votes.cfind(query);
+    allVotes: async (root, {
+      filter, orderBy, limit, skip,
+    }, { db: { Votes } }) => {
+      const query = filter ? { $or: buildVoteFilters(filter) } : {};
+      let cursor = Votes.cfind(query);
       cursor = buildCursorOptions(cursor, orderBy, limit, skip);
-      return await cursor.exec();
+      return cursor.exec();
     },
 
-    syncInfo: async (root, {}, {db: {Blocks}}) => {
+    syncInfo: async (root, {}, { db: { Blocks } }) => {
       let syncBlockNum = null;
       let blocks;
       try {
-        blocks = await Blocks.cfind({}).sort({blockNum:-1}).limit(1).exec();
-      } catch(err){
+        blocks = await Blocks.cfind({}).sort({ blockNum: -1 }).limit(1).exec();
+      } catch (err) {
         console.error(`Error query latest block from db: ${err.message}`);
       }
 
-      if(blocks.length > 0){
+      if (blocks.length > 0) {
         syncBlockNum = blocks[0].blockNum;
       }
 
       let chainBlockNum = null;
       try {
-       let resp = await fetch('https://testnet.qtum.org/insight-api/status?q=getInfo');
-       let json = await resp.json();
-       chainBlockNum = json['info']['blocks'];
-      } catch(err) {
+        const resp = await fetch('https://testnet.qtum.org/insight-api/status?q=getInfo');
+        const json = await resp.json();
+        chainBlockNum = json.info.blocks;
+      } catch (err) {
         console.error(`Error GET https://testnet.qtum.org/insight-api/status?q=getInfo: ${err.message}`);
       }
 
-      return {'syncBlockNum': syncBlockNum, 'chainBlockNum': chainBlockNum }
-    }
+      return { syncBlockNum, chainBlockNum };
```

Comment:
find this syntax confusing

Reply:
it's called shortform in JS. basically if the key and value are the same name, you just need to use it once.